shared/clusterconfig.go

```diff
@@ -167,7 +170,8 @@ func addClusterFromKubeConfig(nodes []Node) (*Cluster, error) {
 // newCluster creates a new cluster and returns his values from terraform config and vars.
 func newCluster() (*Cluster, error) {
 	product := os.Getenv("ENV_PRODUCT")
-	terraformOptions, varDir, err := addTerraformOptions(product)
+	module := os.Getenv("ENV_MODULE")
```

Comment:
dont need to be this PR of course , but we need to fix the passing around of some env vars, since in the middle of runtime we could potentially messed with those, since they are in a psychical fille as .env

Reply:
Sure keeping it as is for now